user-guide/data-sources/bigquery.md

```diff
@@ -0,0 +1,22 @@
+# BigQuery Setup
+
+From your Google Cloud Console, you need to create a service account and assign it permissions to access BigQuery. See details on creating a service account here: https://developers.google.com/identity/protocols/OAuth2ServiceAccount#creatinganaccount
+
+> #### primary::
+>
+> Note that when generating the key file, select the JSON type and later use it when setting > up the data source.
+
+From all the settings the mandatory ones are the key file and the Project ID.
```

Comment:
This feels like it's related to the service account setup, while in fact it's a comment on our settings...
We can add a screenshot of the settings and explain each one of them. Below it to explain service account creation.